docs/markdown/Getting Started/getting-started/prerequisites.md

```diff
@@ -38,25 +33,19 @@ System-specific notes
 > 
 > If you need to run Pants on Alpine, [let us know](doc:community), so we can prioritize this work. Meanwhile, you can try [building Pants yourself](doc:manual-installation#building-pants-from-sources) on Alpine.
 
-> 🚧 Linux on ARM is not yet supported
+> 🚧 Linux on ARM will be supported from Pants 2.16
```

Comment:
Since these are the Pants 2.16 docs, this section could be present tense. Unless you're planning on cherry-picking this change to 2.15...?

Reply:
I was planning on cherrypicking, possibly even to 2.14.x, so we can get this rolling sooner.

Reply:
Hm. I think that given the questions from the meeting, we might want to have it be the recommended approach for installing 2.16 first, and then only cherry-pick to older versions if that goes well? But I don't feel strongly about it.

Reply:
The audience is primarily new users, so why would it matter which version of Pants we target?

Reply:
> The audience is primarily new users, so why would it matter which version of Pants we target?

Because of the volume of users that we get on stable vs unstable versions. Trying this out with non-power users in a low-volume situation (alphas and release candidates for a new release) as opposed to an already stable release might better align with expectations.